<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerAddContent</class>
 <widget class="QWidget" name="LayerAddContent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="wBg" native="true">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>元件</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btnAddText">
           <property name="text">
            <string>文字</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddPic">
           <property name="text">
            <string>圖片</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddVideo">
           <property name="text">
            <string>影片</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddBtn">
           <property name="text">
            <string>按鈕</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddMar">
           <property name="text">
            <string>跑馬燈</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddQr">
           <property name="text">
            <string>QR CODE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddUrl">
           <property name="text">
            <string>URL</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
